research/picai/README.md

```diff
@@ -94,35 +94,82 @@ The raw annotations differ in spatial resolution across annotators. Specifically
 For more information on the raw dataset (imaging or labels) or splits other than the Public Training and Development Dataset, refer to the [PICAI competition documentation](https://pi-cai.grand-challenge.org/DATA/).
 
 ### Preprocessed Dataset
-In addition to to Raw Dataset, a Preprocessed Dataset has been provided on the cluster that was generated using the [picai_baseline package](https://github.com/DIAGNijmegen/picai_baseline). This package was provided by the PICAI competition to give a starting point for practitioners to preprocess the data from the Public Training and Development Dataset and train a simple model for the csPCa task.
+In addition to to Raw Dataset, a Preprocessed Dataset has been provided on the cluster that was generated using scripts described in the following section. The preprocessed dataset has been prepared in the format the [nnUNet framework](https://github.com/MIC-DKFZ/nnUNet) making it easy to apply out of the box to the dataset. For more information about nnUNet as it relates to our experimentation on the PICAI dataset, refer to [nnUNet_overview.md](../picai/nnunet_scripts/nnunet_overview.md). Additional preprocessing utilities and training baselines are available in the [picai_prep package](https://github.com/DIAGNijmegen/picai_prep) and [picai_baseline package](https://github.com/DIAGNijmegen/picai_baseline). These packages were provided by the PICAI competition to give a starting point for practitioners to preprocess the data from the Public Training and Development Dataset and train a simple model for the csPCa task.
```

Comment:
- to to Raw Dataset!
- prepared in the format the [nnUNet Framework](https://github.com/MIC-DKFZ/nnUNet) expects, making...